src/NuGet.Core/NuGet.CommandLine.XPlat/Strings.resx

```diff
@@ -947,8 +947,12 @@ Non-HTTPS access will be removed in a future version. Consider migrating to 'HTT
   <data name="WhyCommand_Message_NonSDKStyleProjectsAreNotSupported" xml:space="preserve">
     <value>Unable to run 'dotnet nuget why' for project '{0}'. This command only works on SDK-style projects.</value>
```

Comment:
Should we update the message here so that customers are aware that to use why command on non-sdk style projects they can pass assets file path? https://github.com/NuGet/NuGet.Client/blob/38ab39a6691e6e5d54e29055d9e258b8b5d57220/src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/WhyCommand/WhyCommandRunner.cs#L90-L93

Reply:
Good find!